.travis.yml

```diff
@@ -16,7 +16,7 @@ git:
 matrix:
   include:
   - rust: stable
-    env: RUN_TESTS="true"
+    env: RUN_TESTS="true" TEST_OPTIONS="--no-release"
```

Comment:
Yes, but only on `stable` compiler. Otherwise travis was timeing out (I was trying to re-run couple of times). Suprisingly it's a bit faster when running without `release` on `stable`
.